### ISSUE TITLE
Is further development of this fork planned?

### ISSUE THREAD

**Issue by asiekierka** [reinvdwoerd/nitrotracker#1]
Hey,

I'm working on my own update/fork of NitroTracker ( https://github.com/asiekierka/libntxm and https://github.com/asiekierka/nitrotracker ). So far, I have focused on adding proper DSi mode support, as well as memory leak fixes and GUI performance improvements.

Are you planning to continue developing your fork? I'd like to either suggest joining forces (if that's the case) or ask for your opinion on porting your patches and improvements over to my codebase (if that's not the case).

**Comment by goudreinette** [comment]
Hi, yes very much so! I was temporarily occupied with the finals of my study, but I’d love to work together on this!

**Comment by goudreinette** [comment]
I think adding DSi support will also make it work through TwilightMenu for the 3DS, currently it freezes.

**Comment by asiekierka** [comment]
Do you have any kind of contact? (IRC/Discord/etc.)

Re: your fork - I think the reason you had issues with the touchscreen is because libnds added a requirement to call touchInit(); in arm7 - at the very least, this fixed your fork for me on melonDS. The reason rawx/rawy values don't match is because they're uncalibrated - the calibration values differ for every console, and perhaps for every emulator too.

(Another issue in DSi mode is that calling fatInitDefault more than once - which libntxm does upon trying to open a .XM file if not fixed - will cause a freeze.)

And yes, I have tested my version on the 3DS, and it works splendidly - after patching out tobmic and using libnds's recent microphone code instead, it even supports 16-bit microphone input!

**Comment by goudreinette** [comment]
Fantastic! I have Discord, perhaps we could set up a server so Tobias can join as well 😊

**Comment by asiekierka** [comment]
For now, I think a private conversation will be sufficient...

**Comment by illettante** [comment]
tossing a coin into the wishing well

**Comment by goudreinette** [comment]
YES! @asiekierka developed it a bit further on their fork, but yes I am committed to the flourishing of nitrotracker

**Comment by illettante** [comment]
I like to hear that! It may just be the best portable tracker workflow around because of the DS hybrid button-and-touch interface. Sunvox and MilkyTracker are the only other real contenders because they run on so many devices, but touch-only devices lose the speed of the tracker interface and most palm devices don't have enough buttons, and/or don't allow them to have full use of the hardware buttons because they're used by the system. The LSDJ-interface family of trackers like LGPTracker and the M8 have very fast workflows but suffer from not having the flexibility a touch/cursor/stylus hybrid brings.

...thank you for coming to my ted talk. I'll follow @asiekierka 's fork too!
